gradle/libs.versions.toml

```diff
@@ -24,8 +23,6 @@ timber = { group = "com.jakewharton.timber", name = "timber", version.ref = "tim
 # Plugins
 android-gradlePlugin = { group = "com.android.tools.build", name = "gradle", version.ref = "agp" }
 android-gradlePluginApi = { group = "com.android.tools.build", name = "gradle-api", version.ref = "agp" }
-kotlin-gradlePlugin = { group = "org.jetbrains.kotlin", name = "kotlin-gradle-plugin", version.ref = "kotlin" }
-squareup-javapoet = { group = "com.squareup", name = "javapoet", version.ref = "javapoet" }
```

Comment:
I don't remember why I added javapoet anymore. I think it was some version conflict, which forced me to specify a version. It was added in PR #103.

Reply:
It's possible, but now it is working fine especially after we sorted some things out.